overlord/configstate/configcore/kernel_test.go

```diff
@@ -358,30 +363,43 @@ func (s *kernelSuite) testConfigureKernelCmdlineHappy(c *C, option []cmdlineOpti
 }
 
 func (s *kernelSuite) TestConfigureKernelCmdlineDangGrade(c *C) {
```

Comment:
not related to the PR, but I would expand all these Dang to Dangerous

Reply:
Changed